k-distribution/include/builtin/ffi.k

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-2019 K Team. All Rights Reserved.
+require "domains.k"
+
+module FFI-SYNTAX
+  imports INT-SYNTAX
+  imports FLOAT-SYNTAX
+
+  syntax FFIType ::= Int | Float | "#void" [klabel(#void), symbol]
```

Comment:
This isn't really what you want. You have made "0" and "0.0" instances of FFIType instead of making "Int" and "Float". Additionally, take a look at http://www.chiark.greenend.org.uk/doc/libffi-dev/html/Primitive-Types.html#Primitive-Types for the actual list of types. As I've said elsewhere, exclude pointers and structs for now.